logrotate.spec.in

```diff
@@ -39,11 +37,7 @@ install -p -m 644 examples/logrotate-default $RPM_BUILD_ROOT/%{_sysconfdir}/logr
 install -p -m 755 examples/logrotate.cron $RPM_BUILD_ROOT/%{_sysconfdir}/cron.daily/logrotate
 touch $RPM_BUILD_ROOT/%{_localstatedir}/lib/logrotate.status
 
-%clean
-rm -rf $RPM_BUILD_ROOT
-
 %files
-%defattr(-,root,root)
```

Comment:
I agree with @glensc on this.  Unless removing the lines solves a real problem (e.g. build failure) somewhere, we should keep them upstream for compatibility with ancient versions of rpm.